source/android/adaptivecards/build.gradle

```diff
@@ -29,6 +31,65 @@ android {
     }
 }
 
+signing {
+    required { gradle.taskGraph.hasTask("uploadArchives")}
+    sign configurations.archives
+}
+
+
+uploadArchives {
+        repositories {
+            mavenDeployer {
+                beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+                copy{
+                    from '.'
+                    into './build/outputs/aar/'
+                    include 'adaptivecards-release.aar.asc'
+                }
+                copy {
+                    from '.'
+                    into './build/poms/'
+                    include 'pom-default.xml.asc'
+                }
+                pom.artifactId ="adaptivecards-android-arm"
+
+                repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2") {
+                    authentication(userName: NexusUserName, password: NexusPassword)
+                }
+                pom {
+                    setGroupId "io.adaptivecards"
+                    setVersion '1.0.0'
+                }
+
+                pom.project {
+                    name 'Android Adaptive Cards Library'
+                    description 'Android Adaptive Cards Lib'
+                    url 'https://github.com/Microsoft/AdaptiveCards'
+
+                    scm {
+                        url  'https://github.com/Microsoft/AdaptiveCards'
+                    }
+
+                    licenses {
+                        license {
+                            name 'MIT'
+                            url 'https://github.com/Microsoft/AdaptiveCards/blob/master/LICENSE'
+                            distribution 'repo'
+                        }
+                    }
+
+                    developers {
+                        developer {
+                            id 'microsoft'
```

Comment:
Does this need to match our groupId from maven?

Reply:
no, there is a group id field in the same file; 'setGroupId'